Sources/ShellOut.swift

```diff
@@ -307,6 +307,16 @@ public struct ShellOutError: Swift.Error {
     public let outputData: Data
     /// The output of the command as a UTF8 string, as returned through `STDOUT`
     public var output: String { return outputData.shellOutput() }
+
+    public var debugDescription: String {
+        return
+            [
+                "ShellOutError",
+                "status  : \(terminationStatus)",
+                "message : \(message)",
+                ]
+                .joined(separator: "\n")
```

Comment:
Wouldn't it be simpler to just write this as a string literal, instead of as a joined array? 🤔